src/uproot/models/TBasket.py

```diff
@@ -272,7 +272,10 @@ def read_members(self, chunk, cursor, context, file):
             cursor.skip(self._members["fKeylen"])
 
             self._raw_data = None
-            self._data = cursor.bytes(chunk, self.border, context)
+            if self.border == 0:
+                self._data = numpy.empty(0, dtype=numpy.uint8)
```

Comment:
Is there a reason to special case this outside of `cursor.bytes`? I.e., is there a time when we'd want to call `cursor.bytes` with `self.border = 0`?

Reply:
The signature for `cursor.bytes` is `chunk, length, context`, so this is a special case for `length=0`.

We could have `cursor.bytes` always short-circuit like this whenever the requested length is zero, but then we wouldn't find out about issues like this one. It was only after I found out about the missing second TKey in this file that I was comfortable ignoring the fact that `cursor` is outside of the `chunk`. If it happens in other circumstances, I'd want to know why because it might be an indication of a bigger misunderstanding about the ROOT format.

That's the only reason.

Reply:
OK, so this isn't fixing a broken `bytes()` for zero bytes, it's that the conditions for using `bytes()` are not met in this context?  Fab.

Reply:
That's right: the real issue here is the

```python
cursor.skip(self._members["fKeylen"])
```

just above it. The whole story is here: https://github.com/scikit-hep/uproot5/issues/750#issuecomment-1270469267.

Reply:
Ah, reading that again, that's exactly what you're saying! Great.